packages/react-swipeable-views-utils/package.json

```diff
@@ -20,7 +20,8 @@
     "prop-types": "^15.6.0",
     "react-event-listener": "^0.6.0",
     "react-swipeable-views-core": "^0.13.7",
-    "shallow-equal": "^1.2.1"
+    "shallow-equal": "^1.2.1",
+    "@material-ui/types": "^4.0.0"
```

Comment:
This seems unrelated 

```suggestion
```

Reply:
@kodai3, are you planning to fix this PR? It's the last of 3 opened ones, I guess @oliviertassinari is waiting for it before a potential merge & publish

Reply:
I support the idea to have the types built in the library directly.
The types package of Material-UI is meant for internal usages so it shouldn't be here. As for the other pull request. This one can be work on independently.